src/test/java/duke/task/EventTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import task.Event;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+
+    @Test
+    public void addEventTest() {
+        Event event = new Event("Event Test 1", LocalDate.parse("2022-01-01"));
+        assertEquals("[E][ ] Event Test 1(at: 01 Jan 2022)", event.toString());
+    }
+
+    @Test
+    public void saveEventTest() {
```

Comment:
Try to follow the naming conventions for test methods as shown below:
`featureUnderTest_testScenario_expectedBehavior()`